syft/pkg/cataloger/rpm/file_cataloger_test.go

```diff
@@ -0,0 +1,106 @@
+package rpm
+
+import (
+	"testing"
+
+	"github.com/go-test/deep"
+	"github.com/stretchr/testify/require"
+
+	"github.com/anchore/syft/syft/file"
+	"github.com/anchore/syft/syft/pkg"
+	"github.com/anchore/syft/syft/source"
+)
+
+func TestParseRpmFiles(t *testing.T) {
+	tests := []struct {
+		fixture  string
+		expected map[string]pkg.Package
+	}{
+		{
+			fixture: "test-fixtures/rpms",
+			expected: map[string]pkg.Package{
+				"abc": {
+					Name:         "abc",
+					Version:      "1.01",
+					Locations:    source.NewLocationSet(),
+					FoundBy:      "rpm-file-cataloger",
+					Type:         pkg.RpmPkg,
+					MetadataType: pkg.RpmdbMetadataType,
+					Licenses:     []string{"MIT"},
+					Metadata: pkg.RpmdbMetadata{
```

Comment:
we should be creating a new distinct `RpmMetadata` even if the type ends up being dup'd:
```golang
type RpmMetadata RpmdbMetadata
```

Reply:
From offline discussion: we will rename `RpmdbMetadata` -> `RpmMetadata`, and make sure to handle decoding of existing SBOMs with a `metadataType` of `RpmdbMetadata`